### ISSUE TITLE
Add informative popup/alerts to pages where needed

### ISSUE THREAD

**Issue by gauravano** [publiclab/plots2#5275]
The process of getting started with Public Lab can be complex considering we use research notes/ wikis/ questions/power tags, etc. 

We can draw some inspiration for this issue from the https://publiclab.org/post page where we show comments on the left side for helping users.

Let's make a list of pages where the user might get confused at first from terminologies, process, etc. 

### Pages:
(Append page route below with short description, if you have also thought about the popover location, text, etc, please add that too. Thanks!)

Also, last year in December, we started off a discussion about the **Website tour** at https://github.com/publiclab/plots2/issues/4110. 

Let's keep that issue in reference here and also think about at which pages tour would be better(like while explaining a process of creating question, maybe). And, where modular tip/popover would look good.

Thanks!

**Comment by gauravano** [comment]
We can tackle this issue by breaking in parts for each page. We also have to keep the designs same across the website, so we may want to start with this issue after the UI revamp Outreachy project.

@publiclab/community-reps @publiclab/plots2-reviewers @publiclab/reviewers

**Comment by IshaGupta18** [comment]
Absolutely, sounds good. Once the new UI design is decided, let's break it
up and begin working on it. Thanks!

**Comment by divyabaid16** [comment]
Along with this, we can also create a special ```help``` option in the navbar and upon clicking this option, we will be redirected to a page that lists all the pages present in the website. 
When we click on a certain page, it will show a well-written document (notes/wikis) for the user on how to access that page or get help if stuck.
In this way, everything can be found under one heading. This concept is similar to getting help from the documentation of any programming language.

We can also implement this idea along with the comments shown on some pages for the help.

**Comment by divyabaid16** [comment]
Some pages that may need assistance:
- [ ] https://publiclab.org/chat (Chat)
- [ ] https://publiclab.org/questions (Ask a question)
- [ ] https://publiclab.org/wiki/requests (Respond to a Request)
- [ ] https://publiclab.org/wiki/kits (Contribute a kit) 

More can be added.

**Comment by jywarren** [comment]
This is great. Also starting to collect some more tutorials on how to use
the site here: https://publiclab.org/wiki/website

**Comment by gauravano** [comment]
Nice idea @divyabaid16!

**Comment by gauravano** [comment]
Actually, @jywarren's referred wiki looks complete help page in itself. So, I think we can just add a link to that wiki with the text help on our website?
